tslint.json

```diff
@@ -10,7 +10,8 @@
     "only-arrow-functions": false,
     "prefer-const": false,
     "member-access": false,
-    "object-literal-sort-keys": false
+    "object-literal-sort-keys": false,
+    "interface-name": [true, "never-prefix"]
```

Comment:
Why do we opt out of prefixing interfaces?

Reply:
Glimmer doesn't use this convention when defining its interfaces. I thought it'd be better to be consistent with them.

Reply:
yep, that makes sense 👍